fares-data-build-tool/create-data
# feat: cfd-331 integrate time restrictions from global settings into main user journey

# Description

-   Integrate time restrictions from global settings into main user journey.

# Testing instructions

-   Make sure you have some time restrictions created
-   Go through the normal user journey and see if they are selected.
-   Test scenario where there are no time restrictions created.

# Type of change
